export_as_latex.ml

```diff
@@ -39,7 +39,7 @@ let proof_to_latex proof =
         ^ "  \\shipout\\box\\proof\n"
         ^ "}\n\n" in
     let start_proof = "\\begin{document}\n\n\\adaptpage{\n\\begin{prooftree}\n" in
-    let proof_lines = Proof.to_latex proof in
+    let proof_lines = Proof.to_latex None proof in
```

Comment:
Whynot `(Some None)`? If the first rule is an exchange ( e.g.: [A*A^,A,A^](https://click-and-collect.linear-logic.org/?s=A*A%5E%2CA%2CA%5E+)), you don't want it to be implicit?

Reply:
The code is a draft with the idea that the current state should not modify the previous behavior: thus latex export is with explicit exchange here.

Reply:
That's a mistake: `None` should stay `None` forever in recursive calls (corresponding to explicit exchange), and similarly `Some` should stay `Some` (corresponding to implicit exchange).